api/service.go

```diff
@@ -339,7 +339,8 @@ func (s *service) Listen(req *pb.ListenRequest, server pb.API_ListenServer) erro
 func (s *service) processCreateRequest(req *pb.ModelCreateRequest, createFunc func(...interface{}) error) (*pb.ModelCreateReply, error) {
 	values := make([]interface{}, len(req.Values))
 	for i, v := range req.Values {
-		values[i] = &v
+		s := v
```

Comment:
Allocate a new memory address, otherwise we end up with a slice of pointers that all point to the same address.